src/Umbraco.Core/UriExtensions.cs

```diff
@@ -142,10 +143,18 @@ internal static bool IsDefaultBackOfficeRequest(this Uri url)
         /// <returns></returns>
         internal static bool IsClientSideRequest(this Uri url)
         {
-            var ext = Path.GetExtension(url.LocalPath);
-            if (ext.IsNullOrWhiteSpace()) return false;
-            var toInclude = new[] { ".aspx", ".ashx", ".asmx", ".axd", ".svc" };
-            return toInclude.Any(ext.InvariantEquals) == false;
+            try
+            {
+                var ext = Path.GetExtension(url.LocalPath);
+                if (ext.IsNullOrWhiteSpace()) return false;
+                var toInclude = new[] {".aspx", ".ashx", ".asmx", ".axd", ".svc"};
+                return toInclude.Any(ext.InvariantEquals) == false;
+            }
+            catch (ArgumentException ex)
+            {
+                LogHelper.Error(typeof(UriExtensions), "Failed to determine if request was client side", ex);

```

Comment:
We should probably make this a `LogHelper.Debug` instead so we don't spam the logs when a bot is trying to find the WordPress login page a million times.